apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -107,7 +107,14 @@ export const sortRows = (data, columnIndexList, orderList) => {
   }
   addRows = orderBy(addRows, columnIndexList, orderList);
   newStudentRows = orderBy(newStudentRows, columnIndexList, orderList);
-  studentRows = orderBy(studentRows, columnIndexList, orderList);
+  // Sort students primarily alphabetically, secondarily by account creation
+  // date which aligns with id # (so that older accounts come first)
+  studentRows = studentRows.sort(function(a, b) {
+    if (a.name === b.name) {
+      return a.id > b.id ? 1 : -1;
+    }
+    return a.name > b.name ? 1 : -1;
```

Comment:
I forgot 1 thing, should we convert names to all uppercase or lowercase before comparing them?

Reply:
Good point, thank you!